run-ib-relval.py

```diff
@@ -10,6 +10,20 @@
 from sys import exit, argv
 from optparse import OptionParser
 from os import environ, system, waitpid
+
+try:
+    from os import waitstatus_to_exitcode
+except ImportError:
+
+    def waitstatus_to_exitcode(status):
```

Comment:
@iarspider , `os` is not imported so calls like ` os.WIFEXITED` will fail . By the way what benefit we get from using `waitstatus_to_exitcode`  ? Note that exit code here are not from the `runTheMatrix/cmsDriver` commands  so basically we do not care much about the exit code . if `jobscheduler.py` fails then we mark jenkins job failed so no need to use ` waitstatus_to_exitcode`